ports/stm32/boards/NUCLEO_G0B1RE/mpconfigboard.h

```diff
@@ -1,6 +1,8 @@
 #define MICROPY_HW_BOARD_NAME       "NUCLEO-G0B1RE"
 #define MICROPY_HW_MCU_NAME         "STM32G0B1xE"
 
+#define MICROPY_QSTR_BYTES_IN_HASH  (0)
```

Comment:
This board actually has a reasonable amount of flash, 352k for the app (512k total).  So I'd suggest not disabling hashing on this board.